logging/README.md

```diff
@@ -239,23 +239,13 @@ The script removes configmaps and secrets that were created by the deployment sc
 
 ### Access Kibana
 
-If the deployment process completes without errors, a message such as this
-appears in the console window:
+If the deployment process completes without errors, a message 
+appears in the console window containing the URL address for Kibana and the URL for Elasticsearch (if you enabled access to Elasticsearch). The URL for Kibana is different depending on whether you use nodeports or ingress to access Kibana. 
```

Comment:
The URL is different depending on how things are configured, it isn't just that there's one URL if nodeports are in use and another if ingress is configured.  So, I think the last sentence in this paragraph can be dropped or it needs to be reworded.  

Also, we probably should clarify that the URL shown for Elasticsearch is really only useful for API access and is not designed to used in a web browser.  Maybe we should refer to it as the "URL for the Elasticsearch API endpoint"?

Reply:
Or just don't display the Elasticsearch URL. It's not necessary in most cases and our output should be reserved for only the most important services - Grafana and Kibana.